api/resources/lang/en/validation.php

```diff
@@ -39,7 +39,7 @@
     'digits_between' => 'The :attribute must be between :min and :max digits.',
     'dimensions' => 'The :attribute has invalid image dimensions.',
     'distinct' => 'The :attribute field has a duplicate value.',
-    'email' => 'The :attribute must be a valid email address.',
+    'email' => 'The :attribute you entered has an invalid format.',
```

Comment:
Such validation messages do not have to be exactly the same. Please change it back to the default. The same for all validation messages.